src/video/wayland/SDL_waylandsym.h

```diff
@@ -84,9 +84,11 @@ SDL_WAYLAND_MODULE(WAYLAND_CLIENT_1_18)
 SDL_WAYLAND_SYM(void, wl_proxy_set_tag, (struct wl_proxy *, const char * const *))
 SDL_WAYLAND_SYM(const char * const *, wl_proxy_get_tag, (struct wl_proxy *))
 
+#if SDL_WAYLAND_CHECK_VERSION(1, 20, 0)
 SDL_WAYLAND_MODULE(WAYLAND_CLIENT_1_20)
```

Comment:
@flibitijibibo, note that #5429 will likely need a bit of adjustment if this one gets applied.

Reply:
I've changed this so that `SDL_VIDEO_DRIVER_WAYLAND_DYNAMIC` guarantees that `SDL_WAYLAND_HAVE_WAYLAND_CLIENT_1_20` exists, and tested with this + #5429. It now works.